testnet/shared/config.yaml

```diff
@@ -11,9 +11,10 @@ CONFIG_NAME: 'lukso'
 # ---------------------------------------------------------------
 # Min Genesis Validators in Genesis state
 MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 4000
-# Date and time (GMT): Monday, 2 May 2023 15:45:00
-MIN_GENESIS_TIME: 1683042356
-# 2400 seconds (40 Min) / 16128 + 1683042356 + 2400 - 2
+# Date and time (UTC): Wednesday, 3 May 2023 16:20:00
```

Comment:
I don't know if we should write it as "the third" or like this. Either ways.

```suggestion
# Date and time (UTC): Wednesday, 3rd May 2023 16:20:00
```

Reply:
`Wednesday, May 3rd` is the correct order I think, but ok for me